docs/sql-distributed-sql-engine-spark-sql-cli.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: global
+title: Spark SQL CLI
+displayTitle: Spark SQL CLI
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+* Table of contents
+{:toc}
+
+
+The Spark SQL CLI is a convenient tool to run the Hive metastore service in local mode and execute SQL
+queries input from the command line. Note that the Spark SQL CLI cannot talk to the Thrift JDBC server.
+
+To start the Spark SQL CLI, run the following in the Spark directory:
+
+    ./bin/spark-sql
+
+Configuration of Hive is done by placing your `hive-site.xml`, `core-site.xml` and `hdfs-site.xml` files in `conf/`.
```

Comment:
does it mean we can even configure a remote HMS?

Reply:
> does it mean we can even configure a remote HMS?

In my experience, we can do like this to add some execution related hive conf and hadoop conf.

Reply:
> The Spark SQL CLI is a convenient tool to run the Hive metastore service in local mode ...

Then this is not accurate. Maybe `run the Hive metastore service (in local mode by default) ...`

Reply:
ping @AngersZhuuuu

Reply:
This is not addressed yet.